doc/admin/conf_ldap.rst

```diff
@@ -85,33 +73,23 @@ Configuring Kerberos with OpenLDAP back-end
        access to *
            by * read
 
-    If the locations of the container and principals or the DNs of
-    the service objects for a realm are changed then this
-    information should be updated.
+    If the locations of the container and principals or the DNs of the
+    service objects for a realm are changed then this information
+    should be updated.
 
- 6. Start the LDAP server as follows::
+ 7. In :ref:`kdc.conf(5)`, make sure the following relations are set
+    in the :ref:`dbmodules` subsection for the realm:
 
-       slapd -h "ldapi:/// ldaps:///"
-
- 7. Modify the :ref:`kdc.conf(5)` file to include LDAP specific items
-    listed below::
-
-       realms
-           database_module
-
-       dbmodules
-           db_library
-           db_module_dir
-           ldap_kdc_dn
-           ldap_kadmind_dn
-           ldap_service_password_file
-           ldap_servers
-           ldap_conns_per_server
+       db_library (set to ``kldap``)
+       ldap_kdc_dn
+       ldap_kadmind_dn
+       ldap_service_password_file
+       ldap_servers
```

Comment:
Isn’t **ldap_kerberos_container_dn** also mandatory to set?

Reply:
Yes.  I have updated the PR to add that to the list.